### PR TITLE
Brp 9

### PR DESCRIPTION
**Pull request by cmedlandho**
# What
Added validation to brp number field in the correct-mistakes journey
validation form of:
9 characters long
1-2: Letters 1st letter R
3: Number or X
4-9: Numbers

# Why
So that caseworkers receive the appropriate information to be able to correctly deal with submissions

# How
Added a regex validation to the field 

# Testing
Added acceptance test to check validation fails

### CODE REVIEW

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "brp-card": {
     "label": "BRP number",
-    "hint": "For example, ZR1000452"
+    "hint": "For example, RZ1000452"
```
Comment (sulthan-ahmed) [inline apps/correct-mistakes/translations/src/en/fields.json:50]: is there a reason why this was changed?

Comment (cmedlandho) [reply apps/correct-mistakes/translations/src/en/fields.json:50]: Yes, apparently BRP numbers always start with an 'R' so it wasn't a good hint!

Comment (sulthan-ahmed) [reply apps/correct-mistakes/translations/src/en/fields.json:50]: ok good stuff then

File: apps/correct-mistakes/fields/personal-details.js [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 module.exports = {
   'brp-card': {
-    validate: ['required'],
+    validate: ['required',
+      {type: 'regex', arguments: /^R\D(\d|X)\d{5}$/gi}
```
Comment (sulthan-ahmed) [inline apps/correct-mistakes/fields/personal-details.js:6]: this has been done before many times? worth asking others I'd say but seems good to me for now as part of this ticket :)

**Review by sulthan-ahmed — APPROVED** [review]

**Pull request merged**